lib/realtime/trip_modification.ex

```diff
@@ -29,6 +29,7 @@ defmodule Realtime.TripModification do
       :service_date,
       :last_modified_time
     ]
+    @derive Jason.Encoder
```

Comment:
I just removed a bunch of `@derive Jason.Encoder` annotations without breaking anything. Are these necessary?

Reply:
this one was unneeded, but the rest are [`a05e410f4938f20dbe67da1dccfb2a8a4cb7364b..bf11cdfe783cf811525acebb49d8a7934716336b`#diff-025e8c9325](https://github.com/mbta/skate/compare/a05e410f4938f20dbe67da1dccfb2a8a4cb7364b..bf11cdfe783cf811525acebb49d8a7934716336b#diff-025e8c9325d66be4be78b1284705c85fd7ba87a26ebaed06a3c38b14cd167c36L32)



```
     ** (EXIT from #PID<0.608.0>) an exception was raised:
         ** (Protocol.UndefinedError) protocol Jason.Encoder not implemented for %Realtime.TripModification.SelectedTrip{trip_ids: ["39-0-0-1"], shape_id: "id-of-the-shape"} of type Realtime.TripModification.SelectedTrip (a struct), Jason.Encoder protocol must always be explicitly implemented.

     If you own the struct, you can derive the implementation specifying which fields should be encoded to JSON:

         @derive {Jason.Encoder, only: [....]}
         defstruct ...

     It is also possible to encode all fields, although this should be used carefully to avoid accidentally leaking private information when new fields are added:

         @derive Jason.Encoder
         defstruct ...

     Finally, if you don't own the struct you want to encode to JSON, you may use Protocol.derive/3 placed outside of any module:

         Protocol.derive(Jason.Encoder, NameOfTheStruct, only: [...])
         Protocol.derive(Jason.Encoder, NameOfTheStruct)

             (jason 1.4.1) lib/jason.ex:164: Jason.encode!/2
             (skate 0.1.0) lib/skate/detours/trip_modification_publisher.ex:90: Skate.Detours.TripModificationPublisher.handle_call/3
             (stdlib 5.1.1) gen_server.erl:1113: :gen_server.try_handle_call/4
             (stdlib 5.1.1) gen_server.erl:1142: :gen_server.handle_msg/6
             (stdlib 5.1.1) proc_lib.erl:241: :proc_lib.init_p_do_apply/3
```

Reply:
Yup - I just played around with it myself and saw the same error. Sorry for the 🎢!